src/vl-wizard-pane.js

```diff
@@ -198,6 +250,20 @@ export class VlWizardPane extends VlElement(HTMLElement) {
             });
         }
     }
+
+    _observeActiveClass(callback) {
+        const observer = new MutationObserver((mutations) => {
+            if (mutations.some(mutation => mutation.target.isActive)) {
+                callback();
+            }
+        });
+        observer.observe(this, {attributeFilter: ['class']});
```

Comment:
Gaan we geen extra events uitsturen als een ongerelateerde klasse werd aangepast?

Reply:
ja, de methode doet meer dan de naam zegt alleszins denk ik

Reply:
> Gaan we geen extra events uitsturen als een ongerelateerde klasse werd aangepast?

Momenteel worden er geen andere classes gezet, maar dat kan in de toekomst wijzigen. Ik heb de code daarom uitgebreid.